src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ title, minlength }) => {
```

Comment:
1. `minLength` is not used.
2. `title` is not descriptive. Change it to `label`.
3. What if the input is not required?

Reply:
I did all except first one